config/default/content_access.settings.yml

```diff
@@ -9,3 +9,5 @@ content_access_roles_gids:
   stanford_student: 8
 _core:
   default_config_hash: 7LkiviGITIpniIOOIzpuxR7r4XzqqYN_cwu0fb0kx1M
+content_access_node_type:
+  hs_private_page: 'a:2:{s:4:"view";a:1:{i:0;s:13:"authenticated";}s:8:"per_node";i:1;}'
```

Comment:
This sure ain't pretty.

Reply:
right!? what were they thinking here. :man_shrugging: